src/argus_htmx/templates/htmx/incidents/_incident_tag.html

```diff
@@ -1,2 +1,2 @@
 {% load argus_htmx %}
-{{ incident|tagvalues:tag|first }}
+{{ incident|tagvalues:field.context.tag.key|first }}
```

Comment:
The reason "tagvalues" doesn't have an underscore is because it is a noun, but django template filters can't have a dash (-) in the name.

You might consider renaming "isackedby" to "ackgroup", or "tagvalues" to "has_tag_key"